Dockerfile

```diff
@@ -95,6 +95,14 @@ COPY docker/start.sh /start.sh
 
 CMD /start.sh
 
+# All-in-one Cadence server with Kafka (~550mb)
+FROM cadence-auto-setup AS cadence-auto-setup-with-kafka
+
+RUN apk add openjdk11
+RUN wget https://archive.apache.org/dist/kafka/2.1.1/kafka_2.12-2.1.1.tgz -O kafka.tgz
```

Comment:
This can diverge from the kafka we use. We may think about using this pattern. https://stackoverflow.com/questions/47615751/docker-compose-run-a-script-after-container-has-started

Reply:
yeah this is not ideal. I may switch to running a separate kafka init container within docker compose